cdap-ui/server/express.js

```diff
@@ -151,6 +151,41 @@ function makeApp (authAddress, cdapConfig, uiSettings) {
 
   });
 
+  // This is used for single click deploy app from Cask Market
+  app.get('/streamApp', function(req, res) {
```

Comment:
Could this be used for anything other than app? Like plugin or just plain jar? Would be good if we have a generic endpoint that will just stream any binary content from market to cdap backend.